packages/xo-web/src/xo-app/backup/file-restore/restore-file-modal.js

```diff
@@ -26,7 +33,8 @@ const diskOptionRenderer = disk => (
 
 const partitionOptionRenderer = partition => (
   <span>
-    {partition.name} {partition.type} {partition.size && `(${formatSize(+partition.size)})`}
+    {partition.name} {defined(PARTITION_TYPE_NAMES[+partition.type], partition.type)}{' '}
```

Comment:
Why is casting `partition.type` necessary?

Reply:
The type of `PARTITION_TYPE_NAMES` keys is number and the type of `partition.type` is string. This value is also cast in the existing code.